Source/Manager/SDKConfiguration/SDKConfiguration.swift

```diff
@@ -36,4 +36,16 @@ public class SDKConfiguration {
     func reset() {
         self.agreementsCache = .default
     }
+
+    /**
+     Set this to debug access token and refreshing requests. If set to true then every successful request
+     will also invalidate the access token so that a refresh will be forces
+
+     Only settable in debug mode
+     */
+    #if DEBUG
+        public var invalidateteAuthTokenAfterSuccessfullRequest = false
```

Comment:
Why cannot this be set to `true` when its set to `true` in `Initialize.init` under debug?

Reply:
Good question. My thought was that then ze clients will get the behavior by default when they are developing and I thought opt-in is nicer for debug stuff.

But ... 🤷‍♂️